components/RecipeForm.js

```diff
@@ -0,0 +1,208 @@
+import { ingredients } from "@/lib/ingredients";
+import { symptoms } from "@/lib/symptoms";
+import { useState } from "react";
+import styled from "styled-components";
+import { getSuggestion } from "@/utils/get-suggestions";
+import { useRouter } from "next/router";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  margin: 0 20px;
+`;
+
+const ListItemSelectedValues = styled.li`
+  display: flex;
+  gap: 2vw;
+  border: solid grey 2px;
+  border-radius: 5px;
+  width: auto;
+  padding: 0 2vw;
+`;
+
+export default function RecipeForm({ onAddRecipe }) {
+  const router = useRouter();
+
+  const [ingredientSuggestion, setIngredientSuggestion] = useState();
+  const [symptomSuggestion, setSymptomSuggestion] = useState();
+
+  function handleIngredientsChange(event) {
+    const userInput = event.target.value;
+    getSuggestion(userInput, ingredients, setIngredientSuggestion);
+  }
+
+  function handleSymptomsChange(event) {
+    const userInput = event.target.value;
+    getSuggestion(userInput, symptoms, setSymptomSuggestion);
+  }
+
+  const [selectedIngredients, setSelectedIngredients] = useState([]);
+
+  function selectSuggestedIngredient() {
+    selectedIngredients.includes(ingredientSuggestion) ||
+      setSelectedIngredients([...selectedIngredients, ingredientSuggestion]);
+  }
+
+  function selectUserIngredient(event) {
+    if (
+      event.key === "Enter" &&
+      !selectedIngredients.includes(event.target.value)
+    ) {
+      setSelectedIngredients([...selectedIngredients, event.target.value]);
+    }
+  }
+
+  function deleteSelectedIngredient(ingredientToBeDeleted) {
+    setSelectedIngredients(
+      selectedIngredients.filter(
+        (ingredient) => ingredient !== ingredientToBeDeleted
+      )
+    );
+  }
+
+  const [selectedSymptoms, setSelectedSymptoms] = useState([]);
+
+  function selectSuggestedSymptom() {
+    selectedSymptoms.includes(symptomSuggestion) ||
+      setSelectedSymptoms([...selectedSymptoms, symptomSuggestion]);
+  }
+
+  function selectUserSymptom(event) {
+    if (
+      event.key === "Enter" &&
+      !selectedSymptoms.includes(event.target.value)
+    ) {
+      setSelectedSymptoms([...selectedSymptoms, event.target.value]);
+    }
+  }
+
+  function deleteSelectedSymptom(symptomToBeDeleted) {
+    setSelectedSymptoms(
+      selectedSymptoms.filter((symptom) => symptom !== symptomToBeDeleted)
+    );
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const userRecipe = Object.fromEntries(formData);
+    userRecipe.ingredients = [...selectedIngredients, userRecipe.ingredients];
+    userRecipe.symptoms = [...selectedSymptoms, userRecipe.symptoms];
+    onAddRecipe(userRecipe);
+    event.target.reset();
```

Comment:
as you push back to your homepage, we suggest to remove line 93, because it is not necessary anymore.